tests/Common/TokensTest.php

```diff
@@ -928,9 +931,6 @@ public function testAllBucketsTokenPermissions(): void
         foreach ($token['bucketPermissions'] as $bucketId => $permission) {
             $this->assertEquals(self::BUCKET_PERMISSION_MANAGE, $permission);
         }
-
-        $client->getBucket($newBucketId);
```

Comment:
Tohle by se nemělo mazat, to je ten test, ne? Že to nespadne na exceptioně...

Reply:
No jo vlastne :)

Reply:
vratil som to